pkg/gameserverallocations/allocator_test.go

```diff
@@ -474,8 +477,8 @@ func TestAllocatorAllocateOnGameServerUpdateError(t *testing.T) {
 	require.NoError(t, a.Run(ctx))
 	// wait for 4 gameservers in the cache
```

Comment:
nit: better to update the comments, similar for line 503

Reply:
Good catch. Updating.